controllers/piggyBanksController.js

```diff
@@ -11,8 +11,8 @@ module.exports = {
         description : `Completed: ${job.id} ${job.title}`
       };
       db.PiggyBank.create(dataSet)})
-      .then(transaction => console.log("Deposit money Successfully."))
-      .catch(err => res.status(422).json(err));
+      .then(transaction => console.log("Deposit money successfully."))
+      .catch(err => console.log(err));
```

Comment:
Does the front-end handle the error correctly if you can't deposit money?? The console.log doesn't tell the client/front-end anything. Only the developers know what the error is. I rather have the res.status(422).json(err) and then in the front end handle the response status of 422 and display some error to the client/front-end.